text/string.go

```diff
@@ -74,7 +88,7 @@ func LongestLineLen(str string) int {
 				maxLength = currLength
 			}
 			currLength = 0
-		} else if !isEscSeq {
+		} else if !isEscSeq && !isEscSeqAlt {
 			currLength += RuneWidth(c)
 		}
 	}
```

Comment:
Can you add unit-tests for the changes? Call the affected functions with text containing an escape sequence with a URL as defined in the spec linked?